external/mbedtls/alt/dhm_alt.c

```diff
@@ -78,7 +79,7 @@
 
 #include "mbedtls/alt/common.h"
 #include "mbedtls/alt/dhm_alt.h"
-#include "../../../os/se/sss/isp_oid.h"
+#include "../../../os/arch/arm/src/s5j/sss/isp_oid.h"
```

Comment:
This header file is vendor-specific.
I don't think this file needs to be included here.

Reply:
This is very important comment, which I forgot to remove. Thanks for the comment.